mungegithub/mungers/sync/issue-sync.go

```diff
@@ -333,3 +349,24 @@ func (s *IssueSyncer) syncPriority(obj *github.MungeObject, priority Priority) e
 	}
 	return nil
 }
+
+// validLabel ensures that the given label exists. An error is logged if a label cannot be determined to be valid.
+func (s *IssueSyncer) validLabel(l string) bool {
+	if labels, err := s.config.GetLabels(); err != nil {
```

Comment:
This might eat through a fair amount of quota, since it's not cached... but createIssue shouldn't be called too often anyways, so it's fine.